### ISSUE TITLE
Is it possible to make CLDR data even smaller ?

### ISSUE THREAD

**Issue by onigoetz** [globalizejs/globalize-compiler#19]
Hello,

We are using Globalize for some time now, but only for messages without plurals, so we only needed `supplemental/likelySubtags` in our application.

Now that we want to use all modules, we also need to add the CLDR data.
And we discovered that loading all required cldr datasets ( `supplemental/likelySubtags`, `supplemental/numberingSystems`, `supplemental/plurals`, `supplemental/ordinals`, `supplemental/currencyData`, `supplemental/timeData`, `supplemental/weekData`, `main/{locale}/numbers`, `main/{locale}/currencies`, `main/{locale}/ca-gregorian`, `main/{locale}/timeZoneNames`, `main/{locale}/dateFields`, `main/{locale}/units` )

We end up with 150K for the main data and 150K per locale.
This is quite big and we feel it's not optimal.

Is there a way to reduce this ? strip some parts of these files that are never used by globalize anyway ?

**Comment by rxaviers** [comment]
Please, could you make this a [stack overflow question](https://github.com/globalizejs/globalize#contributing) instead? Also include more details of your application, e.g., is it a front-end or back-end application? Also include why https://github.com/globalizejs/globalize#compilation-and-the-runtime-modules doesnt help you. Thanks.

**Comment by klihelp** [comment]
@onigoetz Can you share if you received any useful answers for your problem?

**Comment by onigoetz** [comment]
Hello @klihelp Yes sure :) 
I'll dig up how I did it at work on tuesday (monday is off in Switzerland)

If I remember correctly, I was able to divide by four the payload on the CLDR data that is shipped.

Of course, if your environment permits it, I recommend using the solution proposed by @rxaviers ; compiling the translations

**Comment by onigoetz** [comment]
Hi @klihelp I created a Gist with the class we use to generate the subset of CLDR Data :

https://gist.github.com/onigoetz/616fc7426fb6c286667279fab313e9c3

I can't include the full code as it's company specific, but the CLDRFileGetter is a class that takes the cldr files from a Jar, the name is the same as the original dataset.

I hope this helps you. It allowed us to take the dataset from 49.8KB to 13.1KB Gzipped. But we also excluded some parts that we don't use.
